AmahiAnywhere/AmahiAnywhere/Presentation/DashboardPresenter.swift

```diff
@@ -29,7 +29,7 @@ class DashboardPresenter: BasePresenter {
         
         self.view?.updateRefreshing(isRefreshing: true)
         let access_token = LocalStorage.shared.getAccessToken()
-        FetchServers.shared.fetch(access_token: access_token) { (serverResponse) in
+        AmahiApi.shared.getServers(access_token: access_token) { (serverResponse) in
```

Comment:
Why not append the access_token in ApiEndpoint file like this. ` 
static func servers() -> String {
        return "\(ApiConstants.BASE_URL)/paths_to_server/\(?access_token=\(LocalStorage().getAccessToken()!)"
    }`